### ISSUE TITLE
InfraDAO experiencing broken panels after recent upgrade

### ISSUE THREAD

**Issue by alex-pakalniskis** [edgeandnode/graphix#97]
## tldr
Kate from InfraDAO has reported issues with panels not working after Vince's recent update. 

## more details
Warning message shown: `This dashboard depends on Angular, which is deprecated and will stop working in future releases of Grafana.
`

[InfraDAO dashboards link](https://graphix.indexerdao.com/dashboards)

Dashboards with broken panels
* [Main dashboard](https://graphix.indexerdao.com/d/dee4d09d-0dbc-4af2-b4a2-2fcb9ecf35cc/graphix-e28093-main-dashboard?orgId=1)
* [Indexer details](https://graphix.indexerdao.com/d/c3d782e5-f574-4dc3-9908-b33367ced256/graphix-e28093-indexer-details?orgId=1)
* [Deployment details](https://graphix.indexerdao.com/d/c76a3ad9-175c-4ac7-874b-901cd316a2a3/graphix-e28093-deployment-details?orgId=1)

Dashboards with no data
* [Network details](https://graphix.indexerdao.com/d/a3c787cf-b38c-4b5b-bbee-11a15ce5dea7/graphix-e28093-network-details?orgId=1)

Untested
* [Bisect tool](https://graphix.indexerdao.com/d/4zm1Ad6Vz/graphix-e28093-bisect-tool?orgId=1)
